packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props

```diff
@@ -1,40 +1,6 @@
 <Project>
-  <PropertyGroup>
```

Comment:
Moved to target

Reply:
Instead of moving the code, just add import into WinFX targets, if they are not already imported.

OR Is that pattern not preferred?